.github/workflows/update.yml

```
@@ -5,6 +5,8 @@
 name: Update
 
 on:
+  schedule:
+    - cron: "0 11,23 * * *"
```

```suggestion
    - cron: "0 11,23 * * 1-5"
```

Let's be nice and don't run on week-ends (although one can still trigger manually if they need it)